models/necks/fpn.py

```diff
@@ -0,0 +1,126 @@
+import torch.nn as nn
+import torch.nn.functional as F
+from ..utils import ConvModule, xavier_init, constant_init
+
+
+class FPN(nn.Module):
+
+    def __init__(self,
+                 in_channels,
+                 out_channels,
+                 num_outs,
+                 start_level=0,
+                 end_level=-1,
+                 add_extra_convs=False,
+                 normalize=None,
+                 use_gn=False,
+                 activation=None):
+        super(FPN, self).__init__()
+        assert isinstance(in_channels, list)
+        self.in_channels = in_channels
+        self.out_channels = out_channels
+        self.num_ins = len(in_channels)
+        self.num_outs = num_outs
+        self.activation = activation
+        self.with_bias = normalize is None
+
+        if end_level == -1:
+            self.backbone_end_level = self.num_ins
+            assert num_outs >= self.num_ins - start_level
+        else:
+            # if end_level < inputs, no extra level is allowed
+            self.backbone_end_level = end_level
+            assert end_level <= len(in_channels)
+            assert num_outs == end_level - start_level
+        self.start_level = start_level
+        self.end_level = end_level
+        self.add_extra_convs = add_extra_convs
+
+        self.lateral_convs = nn.ModuleList()
+        self.fpn_convs = nn.ModuleList()
+
+        for i in range(self.start_level, self.backbone_end_level):
+            l_conv = ConvModule(
+                self.in_channels[i],
+                self.out_channels,
+                kernel_size=1,
+                normalize=normalize,
+                bias=self.with_bias,
+                activation=self.activation,
+                use_gn=use_gn)
+            fpn_conv = ConvModule(
+                self.out_channels,
+                self.out_channels,
+                kernel_size=3,
+                padding=1,
+                normalize=normalize,
+                bias=self.with_bias,
+                activation=self.activation,
```

Comment:
delete all the activation, in FPN, all the conv layers are not with `activation`